### DISCUSSION TITLE
HRtoVrchat

### DISCUSSION THREAD

**Discussion by subjectemulator** [200Tigersbloxed/UnityMods#4] [Q&A] [answered]
on the fitbit app store it is not available, how do i get it to load on the fitbit i have a versa 2 because when i try to get it from the link it is not available.

**Comment by 200Tigersbloxed** [comment]
Hi! You can find the download for the SDK4 version of FitbitHRtoWS at https://gallery.fitbit.com/details/d16ebb83-f52a-4fc3-9819-0447902a9d95

You'll follow the link on your phone (where the app is located) and install it that way. The app looks visible to me, so if it's unavailable, I'd need more information to diagnose it.

**Comment by 200Tigersbloxed** [comment, marked as answer]
Solved with FitbitHRtoWS v1.5.0 release
